packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -2,219 +2,195 @@
 
-    cursor: not-allowed;
-    background-color: var(--_iui-expandable-block-header-background-color-disabled);
-    border-color: var(--iui-color-border-disabled);
+      background-color: var(--_iui-expandable-block-header-background-color-hover);
 
-    @media (forced-colors: active) {
-      --_iui-expandable-block-expander-icon-fill: GrayText;
-      --_iui-expandable-block-header-caption-color: GrayText;
-      --_iui-expandable-block-header-label-color: GrayText;
-      border-color: GrayText;
+      @media (forced-colors: active) {
+        --_iui-expandable-block-expander-icon-fill: Highlight;
+        --_iui-expandable-block-header-caption-color: Highlight;
+        --_iui-expandable-block-header-label-color: Highlight;
+        border-color: Highlight;
+      }
     }
-  }
-}
 
-/// @arg $type Can be one of: expander, status
-@mixin iui-expandable-block-header-icons($type) {
-  @include mixins.iui-icon-style('m');
-
-  @if $type == 'expander' {
-    width: var(--_iui-expandable-block-expander-icon-size);
-    height: var(--_iui-expandable-block-expander-icon-size);
-    fill: var(--_iui-expandable-block-expander-icon-fill);
-    transform: var(--_iui-expandable-block-expander-icon-transform);
-    transition: fill var(--iui-duration-1) ease-out;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: fill var(--iui-duration-1) ease-out, transform var(--iui-duration-1) ease-out;
+    &[disabled],
+    &:disabled,
+    &[aria-disabled='true'] {
+      --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-disabled);
+      --_iui-expandable-block-header-caption-color: var(--iui-color-text-disabled);
+      --_iui-expandable-block-header-label-color: var(--iui-color-text-disabled);
+
+      cursor: not-allowed;
+      background-color: var(--_iui-expandable-block-header-background-color-disabled);
+      border-color: var(--iui-color-border-disabled);
+
+      @media (forced-colors: active) {
+        --_iui-expandable-block-expander-icon-fill: GrayText;
+        --_iui-expandable-block-header-caption-color: GrayText;
+        --_iui-expandable-block-header-label-color: GrayText;
+        border-color: GrayText;
+      }
     }
-  }
-}
 
-@mixin iui-expandable-block-header-label {
-  display: flex;
-  flex-direction: column;
-  flex: auto;
-  min-width: 0;
-  font-size: var(--iui-font-size-2);
-  color: var(--_iui-expandable-block-header-label-color, inherit);
-}
-
-/// @arg $type Can be one of: label, sublabel
-@mixin iui-expandable-block-header-label-text($type) {
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  transition: color var(--iui-duration-1) ease;
-  @media (forced-colors: active) {
-    transition: none;
-  }
-
-  @if $type == 'sublabel' {
-    font-size: var(--iui-font-size-0);
-    color: var(--_iui-expandable-block-header-caption-color, var(--iui-color-text-muted));
+    > .iui-expandable-block-icon {
+      @include mixins.iui-icon-style('m');
```

Comment:
**_unrelated_** to this PR. You should be able to remove this mixin and instead use `<Icon>` in react. This change can be done in #1354

Reply:
as in wrap the element in `<Icon/>` instead of `<Box/>` and remove the css class?

Reply:
should be similar to `ExpandableBlockEndIcon`

Reply:
okay I have updated this (might have forgotten to push yesterday) + resolved the merge conflicts. also re-tested to make sure things are still working as before

Reply:
hold on css is messed up I'm looking into that